.github/workflows/test.yml

```
@@ -19,6 +19,9 @@ concurrency:
 jobs:
   nextest:
     runs-on: ${{matrix.os}}
+    permissions:
+      contents: read
+      code-quality: write
```

actionlint (1.7.12)

[error] unknown permission scope "code-quality". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)